web/src/js/components/ProxyApp.jsx

```diff
@@ -60,30 +52,16 @@ class ProxyAppMain extends Component {
         return _.clone(this.props.location.query)
     }
 
+    componentWillMount() {
+        this.props.fetchSettings();
+    }
```

Comment:
This is kind of the same "where to put it" question as with the websockets - let's discuss this in #1268 first.